content/en/docs/tutorials/online-training/overview.md

```diff
@@ -66,4 +66,6 @@ Here are some of the sites that offer online training for Kubernetes:
 * [Scalable Microservices with Kubernetes (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
 
 * [Self-paced Kubernetes online course (Learnk8s Academy)](https://learnk8s.io/academy)
+
+* [Platzi Kubernetes Course (Spanish)](https://platzi.com/cursos/k8s/)
```

Comment:
Thanks for your contribution! Unfortunately, linking to tutorials not hosted by the CNCF, the Linux Foundation, or the Linux Academy is no longer allowed. We are in the process of updating the Style Guide with content guidelines (PR #15892). Please also see the discussion on PR #15919.

Reply:
Oof, thanks